index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header>
+            <p class="time"> 9:14 <img class="net" src="C:\Users\Harrison\phone network image.jpg" alt="phone network"></p>
```

Comment:
- [ ] I think it would be better to wrap your logo  with an `a `tag instead of a `p` tag. The logo is on the left and has a link (a tag).This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
### example 👇
```
<header>
    <nav class="nav-bar">
        <a href="" class="nav-bar-logo">Mohammed</a>
        <a href="#" class="nav-bar-burger"><img src="rsc/icons/Icon-Menu.svg" alt=""></a>
    </nav>
<header>

```